test/DataFileUtil_server_test.py

```diff
@@ -615,6 +616,51 @@ def test_pack_zip(self):
             self.assertEqual(set(z.namelist()),
                              set(['inzip1.txt', 'inzip2.txt']))
 
+    def test_pack_large_zip(self):
+
+        filename = 'large_file.txt'
+        tmp_dir = os.path.join(self.cfg['scratch'], 'packlargeziptest')
```

Comment:
It'd probably be faster (and less code) to just generate a 2GB file rather than d/ling one.